docs/self-managed/concepts/elasticsearch-privileges.md

```diff
@@ -3,32 +3,43 @@ id: elasticsearch-privileges
 title: "Elasticsearch privileges"
 ---
 
-If you implement Camunda 8 with Elasticsearch as a service provider, you must configure Elasticsearch with the following [privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html) in mind:
+If you implement Camunda 8 with Elasticsearch as a service provider, the following [privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html) may be required:
 
 ## Cluster privileges
 
-- `monitor` - necessary for health check
-- `manage_index_templates` to create and manage index schema on start up, if they don't already exist in Elasticsearch.
-- _Optional_ `manage_ilm` - required only when ILM is enabled
+- `monitor` - Required to check the Elasticsearch cluster health. This privilege provides read-only cluster operations permissions.
+- `manage_index_templates` - Creates the necessary index templates when Operate is started for the first time, or when updating to a newer version of Camunda 8. Once the index templates are created, you can stop Operate, remove this privilege, and then start Operate again.
```

Comment:
❓ It explicitly mentions Operate, what about Tasklist (and Optimize)?

Reply:
I've gone ahead and updated the file in /docs with a change to reference Operate, Tasklist, and Optimize (instead of only Operate). If this looks good I'll update it in the other files. 👍